.github/workflows/test.yaml

```
@@ -1132,23 +1130,73 @@ jobs:
       - name: Install Dependencies
         run: pnpm install
 
-      - name: test ${{ matrix.platform }}
-        id: run-test
+      - name: test DP1 ${{ matrix.platform }}
+        uses: nick-invision/retry@v2
+        with:
+          timeout_minutes: 60
```

```suggestion
          timeout_minutes: 15
```
Would be better, I think?